lib/main_common.pm

```diff
@@ -2194,11 +2195,9 @@ sub load_security_tests_crypt_core {
     loadtest "fips/openssl/openssl_tlsv1_3";
     loadtest "fips/openssl/openssl_pubkey_rsa";
     loadtest "fips/openssl/openssl_pubkey_dsa";
-    loadtest "console/openssl_alpn";
-
     loadtest "fips/openssh/openssh_fips" if get_var("FIPS_ENABLED");
     loadtest "console/sshd";
-    loadtest "console/ssh_pubkey";
+    loadtest "console/ssh_pubkey" unless is_jeos;
```

Comment:
Not obvious (to me) why it wouldn't work on JeOS

Reply:
It was failing, but cannot find the run. Forgot to double-checked it. Thanks for the reminder.

Reply:
What's the failure?

Reply:
> What's the failure?

Some needle work is required and eventually the ssh password has not worked. There is some clean up done by `sshd`, but as it is not adding much on top of `sshd` I haven't really investigated deeper.

Reply:
Then `rm console/ssh_pubkey.pm`?

Reply:
> Then `rm console/ssh_pubkey.pm`?

If it had been our team's module, I would not hesitate. In this case, I have rather created a https://progress.opensuse.org/issues/96170, so they can decide if they want to rewrite it or decommission it.

Reply:
Ok. I suggest making a PR with the removal, that's the quickest way to get feedback (and it'll have to be done anyway)

Reply:
> Ok. I suggest making a PR with the removal, that's the quickest way to get feedback (and it'll have to be done anyway)

deal, I will do

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13082